test/game_team_collection_test.rb

```diff
@@ -43,4 +43,9 @@ def test_it_can_calculate_season_stats
     skip
   end
 
+  def test_it_can_get_goals_by_season
+    season_info = @game_team_collection.games_by_season(@game_collection, "20132014")
+    assert_equal 220, @game_team_collection.goals_by_season(season_info, 6)
+  end
```

Comment:
Something to note, why is the season id in a string? if this test is passing it's not accurately representing the actual behavior

Reply:
I think the spec harness requires the season as a string.

Reply:
Yeah season_id = string